FirebaseAuthUI/FUIPasswordRecoveryViewController.m

```diff
@@ -121,28 +121,28 @@ - (void)recoverEmail:(NSString *)email {
 
   [self.auth sendPasswordResetWithEmail:email
                              completion:^(NSError *_Nullable error) {
-                               // The dispatch is a workaround for a bug in FirebaseAuth 3.0.2, which doesn't call the
-                               // completion block on the main queue.
-                               dispatch_async(dispatch_get_main_queue(), ^{
-                                 [self decrementActivity];
-
-                                 if (error) {
-                                   if (error.code == FIRAuthErrorCodeUserNotFound) {
-                                     [self showAlertWithMessage:FUILocalizedString(kStr_UserNotFoundError)];
-                                     return;
-                                   }
-
-                                   [self.navigationController dismissViewControllerAnimated:YES completion:^{
-                                     [self.authUI invokeResultCallbackWithUser:nil error:error];
-                                   }];
-                                   return;
-                                 }
-
-                                 NSString *message =
-                                 [NSString stringWithFormat:FUILocalizedString(kStr_PasswordRecoveryEmailSentMessage), email];
-                                 [self showAlertWithMessage:message];
-                               });
-                             }];
+     // The dispatch is a workaround for a bug in FirebaseAuth 3.0.2, which doesn't call the
```

Comment:
This bug has been fixed, there is no need to dispatch async to the main queue here again anymore.

Reply:
Done